crates/composer/src/main.rs

```diff
@@ -0,0 +1,37 @@
+use composer::api::Event;
+use composer::api::DEFAULT_SERVER_ADDRESS;
+use eyre::Result;
+use rodio::{source::Source, Decoder, OutputStream};
+use std::fs::File;
+use std::io::BufReader;
+use std::net::UdpSocket;
+
+fn main() -> Result<()> {
+    color_eyre::install()?;
+
+    let socket = UdpSocket::bind(DEFAULT_SERVER_ADDRESS)?;
```

Comment:
I'm still unsure about using `Udp` instead of `Tcp` for this. 

If we were talking about sending audio packets or any other kind of short, time-limited, missable data I would definitely use Udp. However, if I understand where we are in terms of abstraction when it comes to events, an event could signify a single occurrence of something very bad. For example, a single event packet could mean reporting a catastrophic error that would result in a few seconds of screeching (I may be wrong on this though since I don't have the context of your in-person chats so correct me if I am!).

If I'm on the right ballpark about the potential importance of events though, we shouldn't be missing any, and I think the reliability of TCP would work better here.

Reply:
Right. As agreed personally, we'll continue here with UDP but should look into using reliable alternatives. I filled an issue for that https://github.com/tonarino/acoustic_profiler/issues/15